core/controllers/acl_decorators.py

```diff
@@ -3527,3 +3527,46 @@ def test_can_update_suggestion(
 
     test_can_update_suggestion.__wrapped__ = True
     return test_can_update_suggestion
+
+
+def is_from_oppia_android(handler):
+    """Decorator to check whether the request was send from Oppia Android.
+
+    Args:
+        handler: function. The function to be decorated.
+
+    Returns:
+        function. The newly decorated function.
+    """
+
+    def test_is_from_oppia_android(self, **kwargs):
+        """Checks whether the request was generated from Mailchimp.
+
+        Args:
+            **kwargs: *. Keyword arguments.
+
+        Returns:
+            *. The return value of the decorated function.
+
+        Raises:
+            UnauthorizedUserException. If incoming request is not from a valid
```

Comment:
This docstring seems wrong too (but differently...)?

Reply:
Ah thanks for catching as well, fixed